lib/modem_antenna/modem_antenna.c

```diff
@@ -15,21 +15,27 @@ NRF_MODEM_LIB_ON_INIT(gnss_cfg_init_hook, on_modem_lib_init, NULL);
 
 static void on_modem_lib_init(int ret, void *ctx)
 {
+	int err;
+
 	if (ret != 0) {
 		return;
 	}
 
 	if (strlen(CONFIG_MODEM_ANTENNA_AT_MAGPIO) > 0) {
 		LOG_DBG("Setting MAGPIO configuration: %s", CONFIG_MODEM_ANTENNA_AT_MAGPIO);
-		if (nrf_modem_at_printf("%s", CONFIG_MODEM_ANTENNA_AT_MAGPIO) != 0) {
-			LOG_ERR("Failed to set MAGPIO configuration");
+		err = nrf_modem_at_printf("%s", CONFIG_MODEM_ANTENNA_AT_MAGPIO);
+		if (err) {
+			LOG_ERR("Failed to set MAGPIO configuration (ERR: %d, ERR TYPE: %d)",
+			nrf_modem_at_err(err), nrf_modem_at_err_type(err));
```

Comment:
Nitpicking, but maybe it would be better if `err type` would come before `err`. After all, it depends on `err type` whether `err` is valid or not. Also, fixed the indentation.
```suggestion
			LOG_ERR("Failed to set MAGPIO configuration (ERR TYPE: %d, ERR: %d)",
				nrf_modem_at_err_type(err), nrf_modem_at_err(err));
```

Reply:
Thanks! 
I kept `err` before `err_type` as this is how it is in `nrf_modem_lib.c`, but I agree that it makes more sense to have `err type` before `err`. I might add a commit to change this in `nrf_modem_lib.c`. 

Ok, I will simplify it to print the error directly for the AT command calls. If it turns out to be +CME or +CMS errors for these AT commands we can add a more informative log for that later.